test/unit/lib/classes/plugin-manager.test.js

```diff
@@ -462,7 +461,7 @@ describe('PluginManager', () => {
       const commands = ['foo', 'bar'];
       pluginManager.setCliCommands(commands);
 
-      expect(pluginManager.cliCommands).to.equal(commands);
+      expect(pluginManager.cliCommands).to.deep.equal(commands);
```

Comment:
Let's nott change this test.

In general those old tests, if refactored, they should rather be refactored to `runServerless` usage (see https://github.com/serverless/serverless/tree/main/test#unit-tests). We should not invest time into further mainainace of old tests (unless it helps us in some way, as e.g. getting rid of some dependency)